app/pages/project/index.cjsx

```diff
@@ -27,164 +35,222 @@ SOCIAL_ICONS =
   'youtu.be/': 'youtube'
   'youtube.com/': 'youtube'
 
-counterpart.registerTranslations 'en',
-  project:
-    loading: 'Loading project'
-    disclaimer: "This project has been built using the Zooniverse Project Builder but is not yet an official Zooniverse project. Queries and issues relating to this project directed at the Zooniverse Team may not receive any response."
-    nav:
-      research: 'Research'
-      results: 'Results'
-      classify: 'Classify'
-      faq: 'FAQ'
-      education: 'Education'
-      talk: 'Talk'
-
-ProjectAvatar = React.createClass
-  displayName: 'ProjectAvatar'
-
-  render: ->
-    <PromiseRenderer promise={@props.project.get 'avatar'} then={(avatar) =>
-      <img src={avatar.src} className="avatar" />
-    } catch={null} />
 
 ProjectPage = React.createClass
-  displayName: 'ProjectPage'
```

Comment:
Yep, this is set automatically by the JSX parser when a class is assigned to a variable.